AMPLITUDE_API_KEY

```diff
@@ -0,0 +1 @@
+Since the API key is semi-private, this file is replaced in CI with the real API key.
```

Comment:
the API key is in plain text on every deployed instance right? seems like the reason to replace it in CI is less about it being semi-private than it is about it being configurable ie if somebody used a free-and-open-only distro of exo. Is my understanding correct?

Reply:
Yes, the API key is in plain text. Users could extract it and send bogus data to Amplitude, but it is not a secret key. The best practice is to "make it a little more difficult to misuse", but I'm fine either putting the actual data in this file or updating the placeholder text to be more descriptive. Thoughts?

Reply:
Is it possible to simply replace it with a dev key? This way we'd get isolated analytics for folks working on the product?